source/sync/configure/sync-settings.txt

```diff
@@ -372,8 +491,28 @@ Sync Config Object
    * - | ``queryable_fields_names``
        | Array<String>
      
-     - The :ref:`names of the fields <queryable-fields>` that your client 
-       application can query to determine which data to synchronize.
+     - The :ref:`names of the global queryable fields <queryable-fields>` 
+       that your client application can query to determine which data to 
+       synchronize. These fields are queryable across all synced collections.
+
+   * - | ``indexed_queryable_fields_names``
+       | Array<String>
+     
+     - The :ref:`names of the indexed queryable fields <fs-indexed-queryable-fields>` 
```

Comment:
[nit] names -> name

We only allow one IQF at the moment even though this is an array. This is to allow us flexibility in the future for potentially supporting more than 1  IQF

Reply:
Updated to `name` and added more explicit info that this array may contain one element.